src/main.c

```diff
@@ -68,7 +68,7 @@
  *
  * Applications compiled to be executed along with the bootloader will start at
  * 0x2000 (samd21) or 0x4000 (samd51)
- * The bootloader doesn't changes the VTOR register, application code is 
+ * The bootloader doesn't changes the VTOR register, application code is
```

Comment:
This is a lie: https://github.com/adafruit/uf2-samdx1/blob/master/src/main.c#L143

UF2 does set VTOR.